### PR TITLE
[New Rule] Unusual Sudo Activity

### PR DESCRIPTION
**Pull request by randomuserid**
## Issues

https://github.com/elastic/mechagodzilla/issues/106

## Summary

ML rule to accompany the unusual sudo activity job

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/ml/ml_linux_anomalous_sudo_activity.toml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.5.0"]
```
Comment (devonakerr) [inline rules/ml/ml_linux_anomalous_sudo_activity.toml:3]: 1.6.0?

Comment (randomuserid) [reply rules/ml/ml_linux_anomalous_sudo_activity.toml:3]: they were tested on 1.5.0, so that is the minimum version

Comment (bm11100) [inline rules/ml/ml_linux_anomalous_sudo_activity.toml:3]:
```suggestion
ecs_version = ["1.6.0"]
```

Comment (bm11100) [reply rules/ml/ml_linux_anomalous_sudo_activity.toml:3]: if 1.6.0 doesn't implement any breaking changes, then these should be on the latest version.

**Review by bm11100 — APPROVED** [review]

**Review by bm11100 — APPROVED** [review]

File: rules/ml/ml_linux_anomalous_sudo_activity.toml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+
+[rule.threat.tactic]
+id = "TA0004"
+name = "Privilege Escalation"
+reference = "https://attack.mitre.org/tactics/TA0004/"
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1078"
```
Comment (brokensound77) [inline rules/ml/ml_linux_anomalous_sudo_activity.toml:57]: Would [Abuse Elevation Control Mechanism](https://attack.mitre.org/techniques/T1548/) be a better fit than T1078 here?

Comment (randomuserid) [reply rules/ml/ml_linux_anomalous_sudo_activity.toml:57]: I like it, will change

**Review by devonakerr — APPROVED** [review]

File: rules/ml/ml_linux_anomalous_sudo_activity.toml (hunk 2 above)
Comment (randomuserid) [reply rules/ml/ml_linux_anomalous_sudo_activity.toml]: looks like T1548 is not accepted by the tests

Comment (brokensound77) [reply rules/ml/ml_linux_anomalous_sudo_activity.toml]: Looks like the issue was that you hadn't updated your branch to pull in the ATT&CK updates

**Pull request merged**